bindings/const_generator.py

```diff
@@ -56,52 +56,52 @@ def CamelCase(s):
                     'post': '\n',
                 },
                 {   'regex': r'MODE_.*',
-                    'pre': 'bitflags! {\n\tpub flags Mode : u32 {\n',
+                    'pre': 'bitflags! {{\n\tpub flags Mode : u32 {{\n',
```

Comment:
As I said, I had to add a format at line 406 in order to support producing different enum names with the same rule.

The double curly brace serve only because otherwise the format below would complain that there isn't a closing curly brace. I'm not an expert of python, I don't know if there's a better way, anyhow it doesn't have any effects on the generated files (I tested the generation of all the bindings).